### PR TITLE
Modified README.md, added plotly example

### PR DESCRIPTION
**Pull request by qiaodan97**
For issue Include more visualization possibilities in example enhancement #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,5 +74,29 @@ plt.grid(linestyle='dotted')
 plt.show()
 ```
 
+### Plot with plotly
```
Comment (christinoleo) [inline README.md:77]: Can you also add a line of text saying that plotly is not included in the package's dependencies, so it needs to be installed separately, please?

File: README.md [hunk 2]
```diff
@@ -74,5 +74,29 @@ plt.grid(linestyle='dotted')
...
+from neo_force_scheme import NeoForceScheme
+import plotly.graph_objects as go
+
+dataset = np.random.random((100, 100)) # Some dataset without labels
+labels = np.random.random(100) # Per-row labels
+
+nfs = NeoForceScheme(cuda=True)
+projection = nfs.fit_transform(dataset)
+
+fig = go.Figure(data=go.Scatter(x=projection[:, 0],
```
Comment (christinoleo) [inline README.md]: Just so you know, this inline with data=...(x=... is better if broken as two lines. Not that it matter for this code though, this is fine, just letting you know.

**Review by christinoleo — CHANGES_REQUESTED** [review]
Thanks! I am happy to merge after you include a small line in README as I commented.

File: README.md [hunk 3]
```diff
@@ -78,23 +78,27 @@ plt.show()
 ```Python
 import numpy as np
 from neo_force_scheme import NeoForceScheme
-import plotly.graph_objects as go
+# NOTE: plotly is not is not included in the package's dependencies
+# so that it need to be installed seperately.
```
Comment (christinoleo) [inline README.md:82]: thanks

File: README.md [hunk 4]
```diff
@@ -78,23 +78,27 @@ plt.show()
...
 
 nfs = NeoForceScheme(cuda=True)
 projection = nfs.fit_transform(dataset)
 
-fig = go.Figure(data=go.Scatter(x=projection[:, 0],
-                                y=projection[:, 1],
-                                mode='markers',
-                                marker=dict(
+fig = go.Figure(data=
+                go.Scatter(x=projection[:, 0],
```
Comment (christinoleo) [inline README.md:92]: I actually meant
```python
fig = go.Figure(
             data=go.Scatter(
                        x=projection[:, 0],
                        ...
```
but it's fine

**Review by christinoleo — APPROVED** [review]
